src/View/Helper/HtmlHelper.php

```diff
@@ -346,7 +346,8 @@ public function link($title, $url = null, array $options = [])
     {
         $escapeTitle = true;
         if ($url !== null) {
-            $url = $this->Url->build($url);
+            $url = $this->Url->build($url, $options);
```

Comment:
We should probably whitelist options going into the `build()` right now all the attributes for the link go in which will make it hard for us to add options to build() in the future.

Reply:
Is that something for the 4.x.x release, or should we start doing it now?
And if now, what options are going to be whitelisted?

Reply:
I don't know when options will be added, if ever. As for a whitelist we could start with `fullBase`.

Reply:
I changed my mind. The options for UrlHelper::build() have not changed in a very long time. We can deal with the changes later one.